platform/cavium/cavm-sai.mk

```diff
@@ -1,12 +1,20 @@
 # Cavium SAI
 
+CAVM_SAI_GITHUB = https://github.com/garyachy/OpenXPS/raw/sai_0.9.5/SAI/cavm-sai/
```

Comment:
If, for instance, SAI headers are changed, you won't be able to compile older version of SONiC.
But it's up to you.